testui/support/appium_driver.py

```diff
@@ -411,26 +413,21 @@ def __local_run(url, desired_caps, use_port, udid, log_file):
         if udid is None:
             desired_caps = __set_android_device(desired_caps, device)
         logger.log(f'setting device for automation: {desired_caps["udid"]}')
-        root_dir = (
-            os.path.dirname(
-                os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-            )
-            + "/"
-        )
-        Path(root_dir + "appium_logs").mkdir(parents=True, exist_ok=True)
+        root_dir = os.path.join("./logs", "appium_logs")
```

Comment:
the remove logs will be modified, so that it removes from appium_logs folder as well. I rather keep appium logs in a different folder, or otherwise the parent folder mght get crowded pretty quickly, as it creates a new appium log file each scenario